ui/app/AppLayouts/Wallet/panels/SwapInputPanel.qml

```diff
@@ -264,13 +265,13 @@ Control {
                 readonly property double maxSafeValue: WalletUtils.calculateMaxSafeSendAmount(
                                                            d.maxInputBalance, d.inputSymbol)
                 readonly property string maxSafeValueAsString: maxSafeValue.toLocaleString(
-                                                                   LocaleUtils.userInputLocale, 'f', -128)
+                                                                   amountToSendInput.input.locale, 'f', -128)
```

Comment:
hmm can `LocaleUtils.userInputLocale` and `amountToSendInput.input.locale` be different? Or is this change just test related?

Reply:
Right, they should be the same

Reply:
yes they should be the same, but I needed to be able to set locale from the Unit tests and thats why I did this!

Reply:
ah true... the former is a singleton :)